packages/iov-core/src/signingservercore.spec.ts

```diff
@@ -41,11 +41,18 @@ describe("SigningServerCore", () => {
     fractionalDigits: 9,
     tokenTicker: "CASH" as TokenTicker,
   };
+  const defaultGetIdentitiesCallback: GetIdentitiesAuthorization = async (_, matching) => matching;
```

Comment:
This is good to show positive case.
Can you add tests for eg. filtering out all but one or returning an error?
Likewise for the SignAndPost callback with `true`, `false`, `throw Error`... or specify we should never throw an error if that is what you want.

It would be good to see how errors go across the rest boundary as well.